pkg/extensions/search/common/common.go

```diff
@@ -25,7 +27,13 @@ type TagInfo struct {
 	Timestamp time.Time
 }
 
-func GetImageRepoPath(image string, storeController storage.StoreController) string {
+type ImageInfoByDigest struct {
```

Comment:
Where is this used?

Reply:
This is not used. We use TagInfo instead